### PR TITLE
Fix DS MySQL

### PR DESCRIPTION
**Pull request by ebelter**
When using the MySQL datasource, I ran into a few issues getting it to work. This included updating the dbi data source string, escape char, and offset limit.

### CODE REVIEW

**Comment by ebelter** [comment]
@brummett @tmooney Can you guys take a look? I need MySQL to work to untangle our group form oracle.

If approved can we have another release on CPAN (0.46)?

File: lib/UR/DataSource/MySQL.pm [hunk 1]
```diff
@@ -39,16 +39,7 @@ sub _format_sql_like_escape_string { return }
...
-    my($self, $bx) = @_;
-
-    my $tmpl = $bx->template;
-    if ($tmpl->offset and !defined($tmpl->limit)) {
-        return 0;  # Can't have offset without limit
-    } else {
-        return 1;
-    }
-}
+sub does_support_limit_offset { 0 }
```
Comment (brummett) [inline lib/UR/DataSource/MySQL.pm]: If you changed https://github.com/genome/UR/blob/3573d2f1e49d3b20b1a393cb2d005a2f42eebd5d/lib/UR/DataSource/RDBMS.pm#L1601 to pass in `$rule`, does that mean this commit isn't necessary?

The other place `does_support_limit_offset()` is called already passes it in: https://github.com/genome/UR/blob/3573d2f1e49d3b20b1a393cb2d005a2f42eebd5d/lib/UR/Context/LoadingIterator.pm#L76

**Comment by ebelter** [comment]
@brummett Changes made. This update seems to work so far!

**Comment by brummett** [comment]
+2

**Pull request merged**